stdout of test 'tst830` in directory 'monetdb5/mal` itself:


# 08:49:52 >  
# 08:49:52 >   mserver5 "--config=/ufs/mk/monet5//Linux/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/ufs/mk/monet5//Linux/lib/MonetDB5:/ufs/mk/monet5//Linux/lib/MonetDB5/lib:/ufs/mk/monet5//Linux/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/mk/monet5//Linux/var/MonetDB5/dbfarm"    --set mapi_open=true --set xrpc_open=true --set mapi_port=35798 --set xrpc_port=44801 --set monet_prompt= --trace  --dbname=mTests_src_mal  /ufs/mk/monet5/Linux/mTests/src/mal/tst830.mal
# 08:49:52 >  

# MonetDB server v5.13.0, based on kernel v1.31.0
# Serving database 'mTests_src_mal', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.da.cwi.nl:37997/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-13998/.s.monetdb.37997
# MonetDB/GIS module loaded
# Start processing logs sql/sql_logs version 52200
# Start reading the write-ahead log 'sql_logs/sql/log.4'
# Finished reading the write-ahead log 'sql_logs/sql/log.4'
# Finished processing logs sql/sql_logs
# MonetDB/SQL module loaded
# MonetDB/R   module loaded

Ready.

# 11:35:27 >  
# 11:35:27 >  "mclient" "-lmal" "-ftest" "-Eutf-8" "--host=/var/tmp/mtest-13954" "--port=38078"
# 11:35:27 >  

function user.foo():void;               	#[0] (0)  0 
    pw:str := clients.backendsum("Pmonetdb":str);	#[1] (0) CLTbackendsum 1 <- 2 
    id:oid := clients.addUser("monetdb":str, pw:str);	#[2] (0) CLTaddUser 3 <- 4 1 
catch MALException:str;                 	#[3] (0)  5 
    io.printf("0: %s\n":str, MALException:str);	#[4] (0) IOprintf 6 <- 7 5 
exit MALException:str;                  	#[5] (0)  5 
    id:oid := clients.addUser("test":str, pw:str);	#[6] (0) CLTaddUser 3 <- 8 1 
    io.print(id:oid);                   	#[7] (0) IOprint_val 9 <- 3 
    id:oid := clients.addUser("checkpoint":str, pw:str);	#[8] (0) CLTaddUser 3 <- 10 1 
    io.print(id:oid);                   	#[9] (0) IOprint_val 11 <- 3 
    (u:bat[:oid], n:bat[:str]) := clients.getUsers();	#[10] (0) CLTgetUsers 12 13 
    io.print(u:bat[:oid], n:bat[:str]); 	#[11] (0) IOtable 14 <- 12 13 
    clients.removeUser("checkpoint":str);	#[12] (0) CLTremoveUser 15 <- 10 
    (u:bat[:oid], n:bat[:str]) := clients.getUsers();	#[13] (0) CLTgetUsers 12 13 
    io.print(u:bat[:oid], n:bat[:str]); 	#[14] (0) IOtable 16 <- 12 13 
    clients.setPassword("test":str, pw:str);	#[15] (0) CLTsetPassword 17 <- 8 1 
    clients.setPassword("monetdb":str, pw:str);	#[16] (0) CLTsetPassword 18 <- 4 1 
catch InvalidCredentialsException:str;  	#[17] (0)  19 
    io.printf("1: %s\n":str, InvalidCredentialsException:str);	#[18] (0) IOprintf 20 <- 21 19 
exit InvalidCredentialsException:str;   	#[19] (0)  19 
    clients.checkPermission("monetdb":str, "Pmonetdb":str);	#[20] (0) CLTcheckPermission 22 <- 4 2 
catch InvalidCredentialsException:str;  	#[21] (0)  19 
    io.printf("3: %s\n":str, InvalidCredentialsException:str);	#[22] (0) IOprintf 23 <- 24 19 
    InvalidCredentialsException:str := nil:str;	#[23] (0)  19 <- 25 
exit InvalidCredentialsException:str;   	#[24] (0)  19 
    clients.checkPermission("monetdb":str, "x":str);	#[25] (0) CLTcheckPermission 26 <- 4 27 
catch InvalidCredentialsException:str;  	#[26] (0)  19 
    io.printf("4: %s\n":str, InvalidCredentialsException:str);	#[27] (0) IOprintf 28 <- 29 19 
    InvalidCredentialsException:str := nil:str;	#[28] (0)  19 <- 25 
exit InvalidCredentialsException:str;   	#[29] (0)  19 
    clients.checkPermission("x":str, "Pmonetdb":str);	#[30] (0) CLTcheckPermission 30 <- 27 2 
catch InvalidCredentialsException:str;  	#[31] (0)  19 
    io.printf("5: %s\n":str, InvalidCredentialsException:str);	#[32] (0) IOprintf 31 <- 32 19 
    InvalidCredentialsException:str := nil:str;	#[33] (0)  19 <- 25 
exit InvalidCredentialsException:str;   	#[34] (0)  19 
end user.foo;                           	#[35] (0)  
0: MALException:addUser:user 'monetdb' already exists
[ 1@0 ]
[ 2@0 ]
#--------------------------#
# h	t	t  # name
# void	void	str  # type
#--------------------------#
[ 0@0,	0@0,	"monetdb"  ]
[ 1@0,	1@0,	"test"  ]
[ 2@0,	2@0,	"checkpoint"  ]
#--------------------------#
# h	t	t  # name
# void	void	str  # type
#--------------------------#
[ 0@0,	0@0,	"monetdb"  ]
[ 1@0,	1@0,	"test"  ]
1: InvalidCredentialsException:setPassword:The administrator cannot set its own password, use changePassword instead
3: InvalidCredentialsException:checkCredentials:invalid credentials for user 'monetdb'
4: InvalidCredentialsException:checkCredentials:invalid credentials for user 'monetdb'
5: InvalidCredentialsException:checkCredentials:invalid credentials for user 'x'

# 08:49:53 >  
# 08:49:53 >  Done.
# 08:49:53 >  

